NEWS.md

```diff
@@ -293,6 +293,8 @@
 
 41. `tables()` is faster by default by excluding the size of character strings in R's global cache (which may be shared) and excluding the size of list column items (which also may be shared). `mb=` now accepts any function which accepts a `data.table` and returns a higher and better estimate of its size in bytes, albeit more slowly; e.g. `mb = utils::object.size`.
 
+42. `transpose` gains `list.cols=` argument, [#5639](https://github.com/Rdatatable/data.table/issues/5639). This enables to return output with list columns and avoids promoting type to maximum type. `Factor` columns are still promoted to `character` to avoid copying levels. Thans to @MLopez-Ibanez for the request, and Benjamin Schwendinger for the PR.
```

Comment:
Does copying levels is a problem actually? Is there any piece of code that justify this exception?Underneath we copy pointers to strings on global character cache.

Reply:
@ben-schwen that sounds much more reasonable, I would mention that in NEWS. Also this behavior deserves mention in manual page.